.gitattributes

```diff
@@ -1 +1,2 @@
 **/package-lock.json linguist-generated=true -diff
+/runtime/near-vm/tests/wast/spec/* linguist-detectable=false
```

Comment:
I’d just ignore anything with `**.wast`, `**.wat` and `**.wasm` extensions. Given the nature of the project(s) these files can end up in other places just as well quite easily, leading to this same issue appearing again.

Reply:
that makes sense, done